src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
   environment variables) used by GenevaTraceExporter in Linux.
   ([#925](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/925))
 
+* Fix socket connection issue at startup when UDS mount point is not available
```

Comment:
I would suggest that we capture the expectation from the users' perspective. Maybe something like (to be frank I haven't spent much time polishing the message):

"Changed the behavior of during initialization on Linux. Before this change, the exporter initialization would throw exception if the target Unix Domain Socket does not exist; after this change, the exporter initialization would return success and the exporting background thread will try to establish the connection."

Reply:
Updated change log as per your suggestion. Thank you!